### DISCUSSION TITLE
Duda COORDINAR-PEDIDOS

### DISCUSSION THREAD

**Discussion by MaxAl100** [IIC2133-PUC/2024-1#18] [Tarea 0] [answered]
Tengo una duda respecto a la función coordinar pedidos, dice que hay que transferir los pedidos de la nave con menos pedidos al planeta a la nave con más pedidos al planeta, pero no dice cómo es la transferencia. La idea es pasar hacia el final de los pedidos o después del primer pedido que va al planeta?

**Comment by PabloNavarroY** [comment, marked as answer]
Los pedidos se transfieren todos al final de la cola y en el mismo orden relativo en el que estaban antes.

Te adjunto una imagen con un ejemplo del proceso en la que se marcan con rojo los pedidos que van al planeta especificado.

![image](https://github.com/IIC2133-PUC/2024-1/assets/155672565/3188fc7d-a876-48ca-86f3-5f16a43a1ad5)
